src/components/organisms/Room/Room.tsx

```diff
@@ -73,12 +80,10 @@ const Room = ({ roomName, setUserList, capacity = undefined }) => {
     return () => {
       if (localRoom && localRoom.localParticipant.state === "connected") {
         localRoom.localParticipant.tracks.forEach(function (trackPublication) {
-          trackPublication.track.stop();
+          //@ts-ignore
+          trackPublication.track.stop(); //@debt typing does this work?
```

Comment:
what happens here?

Reply:
The types for LocalTrack does not have a stop method, so either the types are wrong or this doesn't do anything